flag/service/service.go

```diff
@@ -4,7 +4,13 @@ import (
 	"github.com/giantswarm/app-operator/flag/service/kubernetes"
 )
 
+// AppCatalog is a data structure to hold AppCatalog specific command line configuration flags.
+type AppCatalog struct {
```

Comment:
When you look at how the flags and the folders to define them are structured, you notice that the structure here should be somewhere else. This is actually pretty easy to understand as there is already the `Kubernetes` structure as well. Let's not reinvent the wheel here.

Reply:
Sure, I just move it out to another file.